libretroshare/src/pqi/authssl.cc

```diff
@@ -1260,8 +1272,9 @@ int AuthSSLimpl::VerifyX509Callback(int /*preverify_ok*/, X509_STORE_CTX* ctx)
 		return verificationFailed;
 	}
 
-	if ( !isSslOnlyFriend && pgpId != AuthGPG::getAuthGPG()->getGPGOwnId() &&
-	     !AuthGPG::getAuthGPG()->isGPGAccepted(pgpId) )
+    if(isSslOnlyFriend && pgpId !=
```

Comment:
This line breaks compilation, how it end up there? This hunk (line 1275-1277) change nothing apart breaking compilation

Reply:
probably a copy+paste issue. The test of same PGP id is alredy there line 1210